### DISCUSSION TITLE
How to retrive a deleted file

### DISCUSSION THREAD

**Discussion by jrsbarbara** [phcode-dev/phoenix#1613] [Q&A] [answered]
I've deleted an important file from Phoenix Code and it was also removed from my desktop. How can I recover it?

**Comment by abose** [comment]
Is this with browser or desktop builds?

**Reply by abose** [reply]
Deleted files are deleted from disk too, so its mostly not recoverable. If you are on windows, [this may help](https://support.microsoft.com/en-us/windows/recover-lost-or-deleted-files-7bf065bf-f1ea-0a78-c1cf-7dcf51cc8bfc) or https://www.ccleaner.com/recuva

**Reply by jrsbarbara** [reply]
I think it's a desktop. I'm still learning how to use it. I download Phoenix from https://phcode.io/#/home. And installed on my laptop. Sorry, for my ignorance. :(
This is how it looks like. 

![Screenshot 2024-05-10 003344](https://github.com/phcode-dev/phoenix/assets/169426854/ecf20396-1725-4f3f-a9f9-2b08c02bec59)

**Reply by abose** [reply]
Hope you were able to recover the files. Try out the above file recovery apps.

**Reply by abose** [reply, marked as answer]
@jrsbarbara This might be late, but from the next release, the default behaviour of delete will be changed to `move to trash` to prevent future issues like this. 

Webstorm and most editors currently delete the files permanently. So we followed the convention. I hope this change will address issues like this from happening in the future.
